projects/dapr/cert-manager-graduation.md

```diff
@@ -0,0 +1,501 @@
+- Dapr has an complete [contributing guide](https://docs.dapr.io/contributing/), including information general guidance, contributing to docs, SDKs, info on the roadmap etc. This is complemented with a dedicated github repository [dapr/community](https://github.com/dapr/community) makes discovery of information quick and easy for new folks.
+- The project makes sure that its [public meetings](https://github.com/dapr/community?tab=readme-ov-file#communication-and-discord) are accessible, with notes, videos, and easy to find meeting links. 
+
+The following actions were provided to the project that were considered blocking but have since been resolved:
+- TOC reviewers are [recommending](https://github.com/dapr/community/issues/530) some of the STC meetings to be public to give a chance for the STC to interact with the project maintainers/users.
+- [Update the Code of Conduct](https://github.com/dapr/community/issues/538) to reference the CNCF Code of Conduct. Currently the Code of Conduct is adapted from the Contributor Covenant, however one of the incubation criteria is to explicitly reference and use the [CNCF CoC](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+The following recommendations were provided to the project that are non-blocking in the TOC's assessment but should be completed by the project to ensure continued viability of the project:
+
+- TAG Security recommends documenting [Security Self-Assessment](https://tag-security.cncf.io/community/assessments/guide/self-assessment/)
+- TAG Security recommends a [joint assessment](https://github.com/cncf/tag-security/blob/main/community/assessments/guide/joint-assessment.md) as well building on the self-assessment above. The existing work already done in Dapr including the several audits and documentation of the [threat model](https://docs.dapr.io/concepts/security-concept/#threat-model) will come in handy.
+- TAG Contributor Experience recommends adopting some tooling such as [clowarden](https://github.com/cncf/clowarden), [prow & peribolos](https://docs.prow.k8s.io/docs/components/cli-tools/peribolos/), [sheriff](https://github.com/electron/sheriff) or other tool that can manage org membership & permissions in a more transparent manner.
+- TOC reviewers [recommend](https://github.com/dapr/community/pull/540/files) a better definition of a "sub-project" as counting each repo as a sub-project ends up with too many of them. Having a better aggregation will help create named roles like "technical leads" which sets up folks better for going up the contributor ladder. TOC reviewers especially worry about "bus factor", having the next rung of leadership mentored and ready will alleviate the situation.
```

Comment:
In the past we've also guided projects on defining lifecycles for their sub-projects, particularly for sub-projects defined as "core" (packaged with the project) and ancillary (those not packaged or required for the main project to function). We've also suggested adding maturity in that lifecycle (alpha, stable, etc.)

Does Dapr's sub-project structure lend itself to those additional recommendations?